EmmanuelCHdz95-Update README.md

BeaLeydier opened the pull request.

EmmanuelCHdz95 committed:
EmmanuelCHdz95-Update README.md

danielanagar commented:
Emmanuel, your bio looks beautiful! I like how you introduced yourself, provided professional and education sections, as well as areas of expertise and personal interests. I also enjoyed the graphics you used to give your bio some life. Lastly, you successfully included headers, subheaders, a picture, a bulleted list, and links, as stipulated. 

Suggested Changes:

- I think the bullet points in your "Professional Experience" section could be improved through more specificity. You explain _what_ you did in each of your different positions, but I don't know _how_ you did those things. For example, you mention evaluating the cost-effectiveness of SDOH programs in healthcare settings, but you don't explain what tools, if any, you used to do so. Was this an analytical evaluation or was there quantitative coding involved? Without knowing the _how_, I don't know the tangible skills you gained through these experiences that you could reapply in other job settings. 

- I also might recommend tightening some of your sections so readers don't get lost in buzzword-y content. I noticed this specifically in the "Areas of Expertise" section.

Great job overall!

EmmanuelCHdz95 committed:
Update README.md

"Aplicados los comentarios del revisor sobre especificidad de las bullet points."
